tasks/start.yml

```diff
@@ -3,5 +3,5 @@
   service: name=influxdb state=started enabled=yes
 
 - name: wait for influxdb port
-  wait_for: port={{influxdb_client_port}} host={{influxdb_bind_address | default('127.0.0.1') }} state=started timeout=30
+  wait_for: port={{influxdb_client_port}} host={{influxdb_bind_address | default('127.0.0.1') }} state=started timeout=180
```

Comment:
I had to increase this timeout. The influxdb (0.9.1) is starting  100-120 seconds on my devstack environmet.
The timeout in wait_for means max time of waiting. If influxdb starts faster then wait_for will finishe earlier.